roles/tftp-server/tasks/main.yml

```diff
@@ -11,10 +18,27 @@
     remote_src: true
     creates: /srv/tftp/pxelinux.0
 
+- name: create dir for configs
+  file:
+    path: "{{ web_root }}/{{ item }}"
+    state: directory
+    recurse: true
+    # owner:  "{{ user_name }}"
+    # group:  "{{ user_name }}"
+  with_items:
+  - lc
+  - d-i/xenial
```

Comment:
I would like to end up supporting both on the same server.  
I need to know what the $release is. or whatever $it is called.

part of my plan is to include fosdem and CCCs setups.

I am not sure how to manage the permutations of preseed deltas  

I think separate templates will be better. maybe.